docs/diagrams/choonwei/DeleteRecurringSessionSequenceDiagram.puml

```diff
@@ -0,0 +1,76 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":DeleteRecurringSessionCommandParser" as DeleteRecurringSessionCommandParser LOGIC_COLOR
+participant ":AddSessionCommand" as AddSessionCommand LOGIC_COLOR
```

Comment:
Hi, not sure if you have miss this out. But I tink it should be "DeleteRecurringSessionCommand" instead of "AddSessionCommand".

Reply:
Woops, caught slackin 😄 Thanks!